### DISCUSSION TITLE
Creating Collections

### DISCUSSION THREAD

**Discussion by pvperez1** [CollectionBuilder/collectionbuilder.github.io#38] [Ideas]
In some of our projects that used CollectionBuilder, we often change the Location metadata to Collection. Since there is already a map to represent location, we used the location word cloud instead to represent collections (just like how Omeka has items and collections), and items can belong to none, one, or more collections.

So hopefully, in the future, a way to group some items into collections? Currently, the collections are showing as a word cloud but it would be super if they can be displayed as cards with their own metadata for the whole collection.

**Comment by evanwill** [comment]
Thanks for the idea @pvperez1 ! 
Can you point me to an example (collectionbuilder or Omeka) so I can get a better understanding of how it works? 

Essentially like a browse for one column in your metadata, but also being able to add additional description to the values in the column?

I think we have built somewhat similar features into some projects. 
For example, we have done a "Glossary" (a list of terms with a name and description) where each term can be associated with objects--each term gets a page that displays the information and pulls in cards displaying any related items. For example [HJCCC](https://www.lib.uidaho.edu/digital/hjccc/glossary/aizu-hongo.html). We have done the same sort of set up but using "Artist" (rather then glossary term), so you could re-frame the feature in many ways depending on what you want to group together and describe. These are driven by creating another CSV with the "glossary" information. We will try to document that customization at some point.

**Reply by pvperez1** [reply]
Hi Evan, yes! I think the HJCCC example is what we are looking for, thanks! In our projects, here is how we re-purposed the "Location" tab (http://kibo.chr.gov.ph/chr-kibo/collections.html, https://curatory.tech/rizal_hall/exhibits.html). An example from omeka is this one: https://info.omeka.net/showcase/example/dulcimerarchive.html. In omeka, collections can have their metadata along with a featured thumbnail that can represent the collection. But yes, the HJCCC example addresses my concern, thanks!

**Reply by evanwill** [reply]
@pvperez1 I think we will try to get the "Glossary" style feature documented as a customization option--it isn't too hard to set up, and we definitely seem to run into that use case regularly. [tracking Issue](https://github.com/CollectionBuilder/cb-docs/issues/11)
